pt/esbocos/originais/git-ferramenta-metodologica-projetos-historia-1.md

```diff
@@ -304,33 +304,33 @@ No ramo main
 No commits yet
 
 Arquivos não monitorados:
-  (utilize "git add <arquivo>..." para incluir o que será submetido)
+  (utilize "git add <ficheiro>..." para incluir o que será submetido)
 	README.md
 
-nada adicionado ao envio mas arquivos não registrados estão presentes (use "git add" to registrar)
+nada adicionado ao envio mas arquivos não registados estão presentes (use "git add" to registar)
 ```
 
-Vamos entender o que o Git está nos dizendo. Ao passarmos o comando `status` para o Git, ele nos informa a situação atual do repositório. Nesse momento, o Git nos informa que estamos no ramo (ou *branch*) `main`: `No ramo main`. Em seguida nos informa que não existem submissões (*commits*) ainda: `No commits yet`[^1]. Mais abaixo veremos o que são *commits* e sua importância metodológica para nossas pesquisas.
+Vamos entender o que o Git está nos dizendo. Ao passarmos o comando `status` para o Git, somos informados da situação atual do repositório. Neste momento, o Git informa que estamos no ramo (em inglês, *branch*) `main`: `No ramo main`. Em seguida, informa que ainda não existem submissões (em inglês, *commits*): `No commits yet`.[^1] Mais abaixo veremos o que são *commits* e sua importância metodológica para as nossas pesquisas.
 
 Em seguida temos a mensagem: 
 
 ```bash
-Arquivos não monitorados: (utilize "git add <arquivo>..." para incluir o que será submetido)
+Arquivos não monitorados: (utilize "git add <ficheiro>..." para incluir o que será submetido)
 	README.md
-nada adicionado ao envio mas arquivos não registrados estão presentes (use "git add" to registrar)	
+nada adicionado ao envio mas arquivos não registados estão presentes (use "git add" to registar)	
```

Comment:
Tendo em conta o glossário, "Arquivos" poderá ser alterado para "Ficheiros"? Ou no código aparece mesmo como "Arquivos"? Será pela utilização de PT do BR? Se for podemos deixar uma nota no início da lição.

Reply:
Manter em pt_BR

Reply:
Alterado. Obrigada!